networkx/algorithms/minors/contraction.py

```diff
@@ -534,7 +534,7 @@ def contracted_nodes(G, u, v, self_loops=True, copy=True):
 identified_nodes = contracted_nodes
 
 
-def contracted_edge(G, edge, self_loops=True):
+def contracted_edge(G, edge, self_loops=True, copy=True):
```

Comment:
Why the new parameter? It doesn't seem to be used in the function. Maybe a bad merge or something?

Reply:
With this one, I was originally using `contracted_edge` and needed to do in place contractions. That function is just a wrapper for `contracted_nodes` which has the option for returning a copy of the graph or performing the contraction in place. `contracted_edge` is just a wrapper for `contracted_nodes` but it always assumed that we wanted to perform the contraction on a copy of the graph. Since it is just a wrapper, we should have any parameter for the parent function available here as well. Ultimately I realized than for my purposes just using `contracted_nodes` made more sense but I still think this is a good addition.

Looking at this function again, I see that I added the new parameter but never passed it into `contracted_nodes`, so good catch!